docs/developer/payments/refunds.mdx

```diff
@@ -204,3 +211,78 @@ mutation {
 
 To request a refund, you need to provide the `grantedRefundId` field. The mandatory refund details are stored as `OrderGrantedRefund`, so the refund request will be created based on this data.
 The `OrderGrantedRefund.status` field will be updated based on the result of the refund action. The `TransactionEvent`s related to the requested action will be accessible via `OrderGrantedRefund.transactionEvents` field.
+
+### Total remaining grant calculations
+
+The amount specified in the `OrderGrantedRefund` determines the portion of the `order.total` that should be refunded to the customer.
+The `order.totalRemainingGrant` specifies the amount that still needs to be refunded
+to achieve a state where the total charged amount equals the `order.total` minus the granted refund amount.
+
+The `order.totalRemainingGrant` amount is calculated based on the following formula:
+
+```Python
+totalRemainingGrant = totalGrantedRefund - alreadyGrantedRefund
+```
+
+where
+
+- `alreadyGrantedRefund` is the difference between already refunded and overcharged amounts
+  ```python
+  alreadyGrantedRefund = max((totalRefunded - overchargedAmount), 0)
+  ```
+- `totalGrantedRefund` is a sum of all `amount` from all `OrderGrantedRefund`. The maximum value is `order.total`, when the value is above,
+  it's replaced with the `order.total`.
+  ```python
+  totalGrantedRefund = max(sum(orderTotalGranted.amount), order.total)
+  ```
+- `totalRefunded` is a sum of `amount` from all transactions with refunded or pending refunds amount
+  ```python
+  totalRefunded = sum(amountRefunded + amountRefundPending)
+  ```
+- `overchargedAmount` defines the `amount` that has been charged over the `order.total` value. It's calculated
+  as a sum of all processed amounts (excluding cancel amounts) minus the `order.total`.
+
+  ```python
+  processedAmount = sum(amountCharged + amountRefunded + amountAuthorized + amountChargePending + amountRefundPending + amountAuthorizePending)
+
+  overchargedAmount = processedAmount - order.total
+  ```
+
+Below is an example order with multiple transactions and `totalRemainingGrant` at each stage:
+
+1. An order has a total of 100 USD and two `TransactionItem`s: one with a `chargedAmount` of 100 USD and the second with `chargedAmount` of 60 USD.
+   The `totalBalance` would be 60, as `chargedAmount` - `order.total` results in 60. The `chargeStatus` is `OVERCHARGED`.
+2. Adding a granted refund of 10 USD would increase in a `totalBalance` of 10.
```

Comment:
You could mention here, why `totalBalance` is increased. In 1st step you say:
> The `totalBalance` would be 60, as `chargedAmount` - `order.total` results in 60

So I think worth mention that value of grantedRefund has impact on this calculation.